sdks/python/scripts/run_integration_test.sh

```diff
@@ -226,6 +240,9 @@ fi
 
 echo ">>> RUNNING integration tests with pipeline options: $PIPELINE_OPTS"
 echo ">>>   test options: $TEST_OPTS"
+# TODO(udim): Pass $SUITE once migrated to pytest. xunitmp doesn't support suite
+#   names.
```

Comment:
Can you create a jira to track it? And maybe link to pytest epic? it's easy to forget after a while.

Reply:
Added existing JIRA